pkg/graph/schema.graphql

```diff
@@ -729,6 +729,13 @@ input DeleteSystemIntakeGRBPresentationLinksInput {
   systemIntakeID: UUID!
 }
 
+"""
+Data needed to manually end a system intake's GRB review voting
+"""
+input ManuallyEndSystemIntakeGRBReviewAsyncVotingInput {
```

Comment:
Since this is just an ID, maybe it should just take a UUID instead of an input type wrapping it? I see a couple other inputs in the schema doing the same that probably don't need their own type too. Thoughts?

Reply:
I'd say it's ok to go ahead and switch to an id here if you are cool with it. EASI does have some examples of mutations that just take an id for an input (like delete TRBRequestAttendee and a few others)